### PR TITLE
Implemented YAML configuration

### LINKED ISSUE #1: Config file to specify excluded tables and table-level excluded columns
Rather than specify columns as environment variable, I think it'd be nice to support this with a configuration file. This could be stored in version control and keep generation consistent. It would also deal with column name collisions for excludes, i.e. `foos.sensitive_column_name` should be excluded, but `bars.sensitive_column_name` should be included. We could have a global exclusions to cover the way it is current handled by environment variables.

Example:

```yaml
# .fixurator.yml

excluded_models:
  - Foo
  
global_exclusions:
  - ssn
  
model_level_exclusions:
  - Bar:
    - secret
    - sensitive
```

### PR DESCRIPTION
**Pull request by berfarah**
[closes #1]

Example YAML:

```yml
# config/fixturator.yml

exclude_timestamps: false

models:
  - name: Driver
  - name: User
    exclude:
      - secret_attribute
      - ssn
```

### CODE REVIEW

**Comment by Supernats** [comment]
`included_models` and `excluded_models` seem to me in opposition to each other. What is the intended behavior for the following case:

We have the following ActiveRecord models:
  + Apple
  + Banana
  + Citron
  + Durian
  + Elderberry

This is our fixutator file:
```yaml 
excluded_models:
  - Citron

included_models:
  - Apple
  - Banana
```

Which of the five models will have fixtures generated when running the task?

**Comment by berfarah** [comment]
Glad you asked! I think this is a problem in my naming of "included". Internally (in the code) I have it as "only", which is more clear - but I'm not sure how to translate that to the config file.

The only models selected would be those that are included.

If you had this:
```yml
excluded_models:
  - Citron

included_models:
  - Apple
  - Citron
```

it would only generate fixtures for Apple.

**Comment by Supernats** [comment]
Okay, now that I have a better understanding of the intent, I think we should scrap the notion of `excluded_models` altogether. We're only creating a situation where the config file could fight itself internally. If we drop the opt-out model, the opt-in model becomes more clear.

```yaml
models:
  - Apple
  - Banana
  - Citron:
    excluded_attribues:
      - seed_count
```

The `models` key is not required, and could be assumed. I put it here to have a sense of grouping in case global settings are to be added at a later time.

If this is truly meant as an opt-in solution, which I think makes the most sense, especially in our use case, we should double down on that and drop support for opt-out, because I would argue our implementation does not behave as expected.

File: lib/fixturator.rb [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
 
-  def self.call(*args)
-    Generator.new(*args).call
+    def to_proc
+      method(:call)
+    end
+
+    private
+
+    def exclusions_for(model:, config:)
```
Comment (berfarah) [inline lib/fixturator.rb]: Good call, I removed the need for it.

Comment (berfarah) [inline lib/fixturator.rb]: Fixed

**Pull request merged**